src/main/java/dev/sigstore/encryption/certificates/transparency/CTConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
could we add a package info file to the directory pointing to where this is forked from and why?

Reply:
Ah yeah I put a range in there. But pkginfo makes sense